.travis.yml

```diff
@@ -7,8 +13,30 @@ notifications:
     email: false
 addons:
   apt:
+    sources:
+    - ubuntu-toolchain-r-test
     packages:
     - libgmp-dev
+    - libmpfr-dev
+    - libmpc-dev
+    - binutils-dev
+    - g++-4.7
+    - g++-5
+    - gcc-5
+    
+install:
+  - export JULIA_SOURCE_DIR=`pwd`
+  - export TEST_CPP="no"
+
+  - git clone https://github.com/symengine/symengine symengine-cpp
```

Comment:
I don't think you need to explicitly clone this. Because from the logs it seems `julia -e 'Pkg.clone(pwd())'` explicitly fetches the symenigine-master code.
Have you tried by simply setting `export CXX=g++-4.7`?